docs/docsite/rst/community/maintainers.rst

```diff
@@ -41,7 +45,7 @@ Issues for modules are routed to their maintainers via an automated process. Thi
 PR Workflow
 -----------
 
-Automated routing of pull requests is handled by a tool called [Ansibot](https://github.com/ansible/ansibullbot). (You could say that he moooo-ves things around.) 
+Automated routing of pull requests is handled by a tool called [Ansibot](https://github.com/ansible/ansibullbot). (You could say that he moooo-ves things around.)
```

Comment:
Cute, but adding stuff like this to the documentation is creating a localization nightmare further down the road. Please remove.

Reply:
OK, cut the pun.